pages/children.md

```diff
@@ -20,15 +20,41 @@ lots of volunteers on hand to show you what you need to know.
 
 ***
 
-Have a look at these videos to see what we got up to in previous years!
+## Activities
 
-<div><iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/u4IMlpmjbYg?rel=0" frameborder="0" allowfullscreen></iframe></div>
-<br/>
-<div><iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/yerWCR3mppk?rel=0" frameborder="0" allowfullscreen></iframe></div>
+### Intro to Python with Code Club Projects
+
+Learn how to code Python perhaps for the first time in this session with Code
+Club, before embarking on projects this afternoon!
+
+### Do something else in Python...
+
+Take Python programming beyond the screen and control real world physical
+objects. A session for children already comfortable with Python basics.
+
+### Hacking Minecraft for Beginners
+
+Get your first taste of programming the world of Minecraft using Python. Great
+for those new to Python programming.
+
+### Pygame Zero: Build a game with Python
+
+Want to be a games developer? Then use your Python skills to make your very own
+game.
+
+### BBC Micro:bit Mayhem and Dr Who
+
+Bring your BBC micro:bit along or play with one of ours to create a tardis or
+control Minecraft! Fun for kids and adults.
+
+### Python for Parents
+
+Because you have no idea what just happened but now your kids are enthusiastic
+and keep asking questions….
```

Comment:
Well spotted -- looks like this got mangled somewhere between Google Docs, my system clipboard, and vim.  That's actually an ellipsis not an underscore, which renders the next full stop redundant.